docs/get_started/get_started_macos.md

```diff
@@ -0,0 +1,532 @@
+```sh   
+./classification_sample_async -i <INSTALL_DIR>/inference-engine/samples/sample_data/car.png -m <ir_dir>/squeezenet1.1.xml -d MYRIAD
+```
+
+</details>
+
+## <a name="basic-guidelines-sample-application"></a>Basic Guidelines for Using Code Samples and Demo Applications
+
+Following are some basic guidelines for executing the OpenVINO™ workflow using the code samples and demo applications:
+
+1. Before using the OpenVINO™ samples, always set up the environment: 
+```sh
+source /opt/intel/openvino/bin/setupvars.sh
+``` 
+2. Have the directory path for the following:
+- Code Sample binaries located in `~/inference_engine_cpp_samples_build/intel64/Release`
+- Demo Application binaries located in `~/inference_engine_demos_build/intel64/Release`
+- Media: Video or image. See <a href="#download-media">Download Media</a>.
+- Model: Neural Network topology converted with the Model Optimizer to the IR format (.bin and .xml files). See <a href="#download-models">Download Models</a> for more information.
+
+## <a name="syntax-examples"></a> Typical Code Sample and Demo Application Syntax Examples
+
+Template to call sample code or a demo application:
+
+```sh
+<path_to_app> -i <path_to_media> -m <path_to_model> -d <target_device>
+```
+
+With the sample information specified, the command might look like this:
+
+```sh
+./object_detection_demo_ssd_async -i ~/Videos/catshow.mp4 \
+-m ~/ir/fp32/mobilenet-ssd.xml -d CPU
+```
+
+## <a name="advanced-samples"></a> Advanced Demo Use 
+
+Some demo applications let you use multiple models for different purposes. In these cases, the output of the first model is usually used as the input for later models.
+
+For example, an SSD will detect a variety of objects in a frame, then age, gender, head pose, emotion recognition and similar models target the objects classified by the SSD to perform their functions.
+
+In these cases, the use pattern in the last part of the template above is usually:
+
+`-m_<acronym> … -d_<acronym> …`
+
+For head pose:
+
+`-m_hp <headpose model> -d_hp <headpose hardware target>`
+
+**Example of an Entire Command (object_detection + head pose):**
+
+```sh
+./object_detection_demo_ssd_async -i ~/Videos/catshow.mp4 \
+-m ~/ir/fp32/mobilenet-ssd.xml -d CPU -m_hp headpose.xml \
+-d_hp CPU
+``` 
+
+**Example of an Entire Command (object_detection + head pose + age-gender):**
+
+```sh
+./object_detection_demo_ssd_async -i ~/Videos/catshow.mp4 \
+-m ~/r/fp32/mobilenet-ssd.xml -d CPU -m_hp headpose.xml \
+-d_hp CPU -m_ag age-gender.xml -d_ag CPU
+```
+
+You can see all the sample application’s parameters by adding the `-h` or `--help` option at the command line.
+
+
+## Additional Resources
+
+Use these resources to learn more about the OpenVINO™ toolkit:
+
+* [OpenVINO™ Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes)
+* [Introduction to Intel® Deep Learning Deployment Toolkit](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Introduction.html)
```

Comment:
Please convert the links from absolute to relative ones, i.e.:
`[Inference Engine Developer Guide](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Deep_Learning_Inference_Engine_DevGuide.html)` 
-->
`[Inference Engine Developer Guide](../IE_DG_Deep_Learning_Inference_Engine_DevGuide)`